src/pages/support/upgrade/index.mdx

```diff
@@ -0,0 +1,128 @@
+  longer required.
+</Note>
+
+### Upgrading Stack Files
+
+There is one minor change to the inner format of stack files in v1.x, as well as a new naming convention for stack files. While v0.x stack files were named `nitric-mystack.yaml`, v1.x stack files are named `nitric.mystack.ts` (replacing the first dash with a dot). The only change to the inner format of the files is removal of the `name` key, which is no longer required or supported. The name of the stack will be now always be inferred from the filename, to avoid confusion and naming conflicts.
+
+### Upgrading Nitric APIs
+
+{/* Move security definitions */}
+
+### Upgrading Nitric Functions
+
+Nitric Functions have been renamed to Nitric Services. While in practice this change is mostly cosmetic, it serves to disambiguate the different elements of a Nitric application. Each services is an entrypoint to a Nitric application, and typically represents a single deployed container image. These images could always include multiple functions, which is where confusion could arise.
+
+### Upgrading Queues
+
+Nitric Queues have been simplified in v1.x. They also include some changes in nomenclature. The `send` method has been renamed to `enqueue` and the `receive` method has been renamed to `dequeue`. While the messages sent to queues were typically referred to as "tasks" in v0.x, they are now referred to as "messages" in v1.x.
+
+Lastly, messages sent to queues used to impose common properties such as `id` and `payloadType`. These properties are no longer imposed by the Nitric SDK, and you're free to reintroduce them as you see fit or leave them out entirely.
+
+### Upgrading Topics
+
+As with Queues, Nitric Topics have also been simplified in v1.x. The messages sent to topics were typically referred to as "events" in v0.x, they are now referred to as "messages" in v1.x. They also no longer automatically include or generate an `id` property or other properties such as `payloadType`. As with Queues, you're free to reintroduce these properties as you see fit or leave them out entirely.
+
+### Changing from Collections to Key/Value
+
+The most significant change in Nitric v1.x is the change from Collections to Key/Value. Collection resources are no longer supported or deployed by Nitric, instead they've been superseded by Key/Value resources. Nitric always strives to provide the most consistent experience across all supported cloud providers, and this change was made to better reflect the capabilities of the underlying cloud providers, while also greatly improving the breadth of options available to fullfil the underlying requirements.
+
+The primary differences between Collections and Key/Value are:
+
+- Key/Value stores do not support querying, and are instead designed to be used as a efficient key/value store.
+- Key/Value stores do not support sub-collections.
+- For Azure deployments Key/Value stores are backed by Azure Table Storage, instead of the MongoDB deployments used for Collections.
```

Comment:
Potentially add a note describing the benefits of moving to Azure Table Storage, i.e. reduced deployment time, reducing costs, scale to zero (?).